### DISCUSSION TITLE
为什么频繁请求：o1098464.ingest.sentry.io？是否可以进行关闭处理？

### DISCUSSION THREAD

**Discussion by RiftRays** [whyour/qinglong#2054] [Q&A]
```
import * as Sentry from '@sentry/react';
import { Integrations } from '@sentry/tracing';
import { loader } from '@monaco-editor/react';

export function init(version: string) {
  // sentry监控 init
  Sentry.init({
    dsn: 'https://3406424fb1dc4813a62d39e844a9d0ac@o1098464.ingest.sentry.io/6122818',
    integrations: [
      new Integrations.BrowserTracing({
        shouldCreateSpanForRequest(url) {
          return !url.includes('/api/ws') && !url.includes('/api/static');
        },
      }),
    ],
    release: version,
    tracesSampleRate: 0.1,
    beforeBreadcrumb(breadcrumb, hint?) {
      if (breadcrumb.data && breadcrumb.data.url) {
        const url = breadcrumb.data.url.replace(/token=.*/, '');
        breadcrumb.data.url = url;
      }
      return breadcrumb;
    },
  });

  // monaco 编辑器配置cdn和locale
  loader.config({
    paths: {
      vs: 'https://cdn.staticfile.org/monaco-editor/0.33.0/min/vs',
    },
    'vs/nls': {
      availableLanguages: {
        '*': 'zh-cn',
      },
    },
  });
}
```

![image](https://github.com/whyour/qinglong/assets/60778141/60e9be1f-c4a2-4485-aff2-f8fa56e9370f)

**Comment by doyoman** [comment]
有办法解决吗？简直被这个请求刷屏了，用旧版2.11.3也是一样

**Comment by hpwater** [comment]
密密麻麻的，3秒钟一次查询！这是什么东西来的？

**Comment by cqd5686** [comment]
这是我找到的解决办法，证实有效
docker exec $(CONTAINER_ID) sed -i '/Sentry\.init/ s/^/return;/' /ql/static/build/loaders/sentry.js

**Reply by donald001** [reply]
执行这个命令后需要重启吗？

**Reply by cqd5686** [reply]
你用AI帮你解析一下这个命令就明白了

**Reply by longlife666** [reply]
实测可用，查了几个方法这个最简单了，谢谢分享

**Reply by sobigworm** [reply]
直接使用上面的命令是无效的，需要先使用docker ps查询青龙的CONTAINER_ID
然后替换$(CONTAINER_ID)
比如：docker exec d1e2820006e4 sed -i '/Sentry\.init/ s/^/return;/' /ql/static/build/loaders/sentry.js

以上是给小白看的教程，老鸟请无视

**Comment by kilvn** [comment]


**Reply by cqd5686** [reply]
配置规则还是会产生大量日志

**Reply by cqd5686** [reply]
自欺欺人的作法，依旧在几秒发送一次请求。就是青龙的问题，上面的解决办法仔细看看吧。

**Comment by sulele89757** [comment]
https://sentry.io/welcome/
Sentry is a developer-first error tracking and performance monitoring platform.
错误跟踪和性能监控平台，可能作者想收集一些报错和性能相关指标。但是太频繁了

**Comment by koboyu123** [comment]
docker exec $(CONTAINER_ID) sed -i '/Sentry.init/ s/^/return;/' /ql/static/build/loaders/sentry.js
将 $(CONTAINER_ID) 更改为你的青龙容器ID，在控制台运行这段命令，然后重启容器
别人分享的方法，实测可用
